package.json

```diff
@@ -149,7 +149,7 @@
     "@types/gulp-sourcemaps": "0.0.35",
     "@types/gulp-util": "3.0.36",
     "@types/jest": "29.5.5",
-    "@types/jest-axe": "3.5.5",
+    "@types/jest-axe": "3.5.9",
```

Comment:
this is starting to look more and more suspicious. can you confirm that there were no public API changes in jest-axe, so we can consider type versioned under 3.x.x as reliable enough ?

Reply:
<img width="435" alt="image" src="https://github.com/microsoft/fluentui/assets/35366351/d65fad98-1892-4b44-93f1-6b9d2e437669">

There are no specific API changes mentioned in the changelog

The code diff of index.js doesn't indicate any API changes
https://github.com/NickColley/jest-axe/compare/v6.0.1...v8.0.0?diff=split&w=

Reply:
ty for double checking !